Templates/GitHubActionsPipeline/README.md

```diff
@@ -79,52 +74,78 @@ It is a combination of GitHub Actions pipeline definitions supporting various wo
- - [actions/tagging_createreleasecandidate/action.yml](./.github/actions/tagging_createreleasecandidate/action.yml)
-    - A composite action, used to create the release candidate, and calculate the release version.
- - [actions/tagging_createproductionreleasetag/action.yml](./.github/actions/tagging_createproductionreleasetag/action.yml)
+ - [actions/tagging_createproductionreleasetag/action.yml](./actions/tagging_createproductionreleasetag/action.yml)
     - A composite action, used to create the production release. 
 
-![pipelineDiagram](pipelineDiagram.png)
+![pipeline-diagram](pipeline-diagram.png)
 
 
 Please review the pipeline definitions to understand the various triggers for which these pipelines may be executed, and also the conditions when stages, jobs, or steps are executed. 
 
-Please make yourself familiar with the [Git branching for mainframe development](https://ibm.github.io/z-devops-acceleration-program/docs/branching/git-branching-model-for-mainframe-dev/#characteristics-of-mainline-based-development-with-feature-branches) documentation. 
+Please make yourself familiar with the [Git branching for mainframe development](https://ibm.github.io/z-devops-acceleration-program/docs/git-branching-model-for-mainframe-dev/#characteristics-of-mainline-based-development-with-feature-branches) documentation. 
+
+### Intended Personas and Seperation of Responsibility 
+
+This pipeline template is intended to be used by two distinct personas, an application developer and a release manager. 
+
+The application developer is intended to make use of the [pipeline controller](./workflows/pipelineController.yml), and has access to the [preview](./workflows/Preview.yml), [preview-zBuilder](./workflows/Preview-zBuilder.yml), [feature](./workflows/Feature.yml), [build](./workflows/Build.yml), and [release](./workflows/Release.yml) pipelines.
+
+Excluding the preview and preview-zBuilder pipelines, all pipelines triggered by the pipeline controller will upload binaries to an artifact manager. Additionally, the only environment that the application developer can deploy to with the pipeline controller is the integration environment. 
+
+
+The release manager is intended to make use of the [deploy](./workflows/Deploy.yml) pipeline, which will retrieve binaries from the artifact manager and deploy to a chosen environment. This includes any testing environment such as an acceptance environment, or production. 
+
+Importantly, the application developer persona is not intended to have access to the [deploy](./workflows/Deploy.yml) pipeline. The release manager may or may not have access to the [pipeline controller](./workflows/pipelineController.yml). 
+
+#### Workflow Diagram
+
+![workflow-diagram](workflow-diagram.PNG)
 
 ### Pipeline variables 
 
 In a default setup, the mainline pipeline is triggered for each new commit. 
 
 When manually requesting a pipeline through the pipeline controller, the following variables can be overwritten:
```

Comment:
Can we call out that these are the manual inputs for the pipeline controller using a heading / or highlighting this in bold, please?